packages/cli/test/scripts/create.test.js

```diff
@@ -7,7 +7,7 @@ import { Contracts, Logger } from 'zos-lib';
 import add from '../../src/scripts/add.js';
 import push from '../../src/scripts/push.js';
 import createProxy from '../../src/scripts/create.js';
-import linkLib from '../../src/scripts/link.js';
+import linkLibs from '../../src/scripts/link.js';
```

Comment:
I would stick to just `link`

Reply:
Agree. Let's leave this up until we do the stdlib naming refactor.